.github/workflows/main.yml

```diff
@@ -50,3 +59,9 @@ jobs:
         working-directory: src/test/resources/docker
         run: docker-compose -f docker-compose.cockroachdb.yml down --volumes
 
+      - name: Archive cockroachdb test results
```

Comment:
Are we good for now to have reports only for MSSQL and cockroach DB? Should we create ticket for re-organising workflow to run all tests in parallel?

Reply:
@KushnirykOleh : Yes, we need to have a ticket to reorganize the workflow. I was going to make changes here but then I decided that it would be best to do it under a separate ticket. This PR can be merged as is -- the improvements to the workflow will become independent.